File: test/demo/wslc-polyglot-hello/polyglot/servers/Srv.java
import java.net.*; import java.io.*;
public class Srv { public static void main(String[] a) throws Exception {
  int p=Integer.parseInt(a[0]); try(ServerSocket ss=new ServerSocket(p)){ Socket c=ss.accept();
    c.getInputStream().read(new byte[64]); c.getOutputStream().write("hello world from [java]\n".getBytes()); c.getOutputStream().flush();
    c.getInputStream().read(new byte[64]); c.close(); } } }

File: test/demo/wslc-polyglot-hello/polyglot/servers/Srv.kt
import java.net.ServerSocket
fun main(a:Array<String>){ val p=a[0].toInt(); ServerSocket(p).use{ ss-> val c=ss.accept()
  c.getInputStream().read(ByteArray(64)); c.getOutputStream().write("hello world from [kotlin]\n".toByteArray()); c.getOutputStream().flush()
  c.getInputStream().read(ByteArray(64)); c.close() } }
